src/util/audit.c

```diff
@@ -100,11 +101,23 @@ int util_audit_drop_permissions(uint32_t uid, uint32_t gid) {
  *
  * Return: 0 on success, or a negative error code on failure.
  */
-int util_audit_log(const char *message, uid_t uid) {
-        int r;
+int util_audit_log(int type, const char *message, uid_t uid) {
+        int r, audit_type;
+
+        switch(type) {
+                case AUDIT_TYPE_AVC:
+                        audit_type = AUDIT_USER_AVC;
+                        break;
+                case AUDIT_TYPE_SELINUX_ERROR:
+                        audit_type = AUDIT_USER_SELINUX_ERR;
+                        break;
+                case AUDIT_TYPE_NOAUDIT:
+                default:
+                        audit_type = 0;
```

Comment:
We don't indent the switch-cases, so this can be reduced by 8ch. And we usually break the default-clauses as well.